network/fakepulsar/fakepulsar_test.go

```diff
@@ -23,22 +23,67 @@ import (
 	"time"
 
 	"github.com/insolar/insolar/core"
+	"github.com/insolar/insolar/testutils/network"
 	"github.com/stretchr/testify/assert"
 )
 
-func onPulse(ctx context.Context, pulse core.Pulse) {
-}
-
+// TODO: write adequate tests instead of this imitation of work
```

Comment:
If test are not adequate lets reject this PR. If they are adequate let's remove this TODO.